### ISSUE TITLE
i cant get it to work on ksp 1.12.5 

### ISSUE THREAD

**Issue by jame2312** [RemoteTechnologiesGroup/RemoteTech#849]

**Comment by tedrown** [comment]
agree

**Comment by JSchneidler** [comment]
Agreed. There's no ground station at KSC, so I cannot launch unmanned from there. Everything else seems to work. Can anyone else confirm this?

**Comment by Kamik423** [comment]
I fixed this by deleting all mods, booting KSP with only Remote-Tech (from CKAN), and then re-installing all other mods using CKAN.
I think this is related to some mysterious missing dependency in manual installation – that is not missing with CKAN.
